spec/controllers/orders_controller_spec.rb

```diff
@@ -0,0 +1,18 @@
+
+require 'rails_helper'
+
+describe  OrdersController  do
+
+  describe "GET 'index'" do  	
+ 
+    it "returns http success" do
```

Comment:
Please use singular quotes here. Do you use rubocop to check the accordance of your code to the [styleguide](https://github.com/bbatsov/ruby-style-guide)?